docs/product/insights/mobile/mobile-screens/screen-loads.mdx

```diff
@@ -35,32 +35,25 @@ Sentry tracks TTID automatically, but [TTFD](/product/insights/mobile-vitals/#ti
 
 By default, the **Screen Loads** page displays metrics for the two releases with the highest screen counts for the time range you’ve selected. To choose a different set of releases to compare, use the “release selector” at the top of the page.
 
-The charts section shows a comparison of TTID and TTFD for your top five screens across your two selected releases.
-
-The sortable table below shows a list of your application's screens with the average TTID and TTFD for each release.
+The charts section shows a comparison of TTID and TTFD metrics for your two selected releases over time.
 
 **Reasons Why You Might Not Be Seeing Any Data:**
 
 - You don’t have any transactions with op `ui.load`
 - Your SDKs don’t meet the minimum SDK requirements
 
-## Screen Summary Page
-
-![Example of Screen Summary](./img/screen-loads-screen-summary.png)
-
-To get additional information about any of your application's screens, click on them to get to the **Screen Summary** page. Here, you’ll see a breakdown of TTID and TTFD values broken out by [device class](/concepts/search/searchable-properties/#device-classification) (high, medium, low, or unknown), across your selected releases. This will help you understand how users with different device performance levels are being affected.
 
-The Screen Summary page also shows the average TTID, average TTFD, and Count (the number of times a screen has been called) - broken out by release.
+ Here, you’ll see a breakdown of TTID and TTFD values broken out by [device class](/concepts/search/searchable-properties/#device-classification) (high, medium, low, or unknown), across your selected releases. This will help you understand how users with different device performance levels are being affected.
```

Comment:
```suggestion
Here, you’ll see a breakdown of TTID and TTFD values by [device class](/concepts/search/searchable-properties/#device-classification) (high, medium, low, or unknown), across your selected releases. This will help you understand how users with different device performance levels are being affected.
```